docs/security.html

```diff
@@ -2366,4 +2374,4 @@ <h3 class="anchor-heading"><a id="security_rolling_upgrade" class="anchor-link">
 security.inter.broker.protocol=SSL</code></pre>
 </script>
 
-<div class="p-security"></div>
```

Comment:
I still do not see why this PR contains this final change. The changes should all be in the ACL section.